src/content/docs/get-started/index.mdx

```diff
@@ -201,7 +165,23 @@ Make sure you are signed into your GitHub account before you start. Otherwise, y
 
 <Task>
 
-### Add Code Sync GitHub app
+### Link repo to content
+
+Edge Delivery Services requires configuration about your site to know what to render. These include at minimum a content reference and a code reference. The simplest approach is to create those references with `fstab.yaml` but you can onboard to Config Service and define the references there if you like.
```

Comment:
Suggest telling them to use `fstab.yaml` and then maybe a note about using the Config Service and why you might want to use that option. -- Oh the note is already there.

```suggestion
Edge Delivery Services requires site configuration details to know what to render. These include at minimum a content reference and a code reference. The simplest approach is to create those references in the `fstab.yaml` configuration file.
```